tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -6841,7 +6841,6 @@ node_modules/chrome-devtools-frontend/front_end/layer_viewer/LayerDetailsView.js
 node_modules/chrome-devtools-frontend/front_end/layer_viewer/LayerDetailsView.js(198,65): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/layer_viewer/LayerDetailsView.js(199,53): error TS2339: Property 'createChild' does not exist on type 'Element'.
 node_modules/chrome-devtools-frontend/front_end/layer_viewer/LayerDetailsView.js(200,52): error TS2555: Expected at least 2 arguments, but got 1.
-node_modules/chrome-devtools-frontend/front_end/layer_viewer/LayerDetailsView.js(239,3): error TS2322: Type 'symbol' is not assignable to type 'string'.
```

Comment:
this is caused by incorrect JSDoc:

```js
/** @enum {string} */
/** @enum {symbol} */
LayerViewer.LayerDetailsView.Events = { PaintProfilerRequested: Symbol('PaintProfilerRequest') }
```

Before the change to use only the immediately preceding jsdoc, we used the first, incorrect jsdoc. Now we use the second, correct one.